### DISCUSSION TITLE
RDA properties with range Work, Expression, Manifestation, Item -- are any external classes valid values??

### DISCUSSION THREAD

**Discussion by briesenberg07** [uwlib-cams/sinopia_maps#87] [seeking feedback]
**DETAILS**  

If the element [has related work of manifestation](http://rdaregistry.info/Elements/m/P30265) is desired for use in a RDA Manifestation template for print monographs, are there any QA sources which might provide acceptable IRI values? Some examples:
- OCLCFAST Uniform Title: Resources typed as [schema:CreativeWork](https://schema.org/CreativeWork), see for example [Adventures of Huckleberry Finn (Twain, Mark)](http://id.worldcat.org/fast/1356110)
- Wikidata lookup: Could be used to retrieve instances of a given item (WD doesn't have 'classes' but has items which are instances of other items) such as [intellectual work](http://www.wikidata.org/entity/Q15621286) and [creative work](http://www.wikidata.org/entity/Q17537576), which might be evaluated for compatibility with range RDA Work.

Or to state the question in another way, *are any resources not explicitly typed as [RDA Works](http://rdaregistry.info/Elements/c/C10001) acceptable as values for properties with range RDA Work in well-formed RDA/RDF??*

CC @pan-zhuo

**Comment by briesenberg07** [comment]
**RELATED QUESTIONS**  

- Similar questions could be asked for RDA properties having other RDA Entities as range.  
- Some MARC2RDA discussions such as the [IRI RWO Discussion](https://github.com/uwlib-cams/MARC2RDA/discussions/384) may apply to use of resources typed using non-RDA classes for RDA properties having other RDA Entities (Agent, for example) as range.

**Reply by briesenberg07** [reply]
**From above:**
> Similar questions could be asked for RDA properties having other RDA Entities as range.  

That is, for example, could a Wikidata resource typed as [human/Q5](http://www.wikidata.org/entity/Q5) be used as the value for an RDA property with range RDA Person in well-formed RDA/RDF?

I'm taking @GordonDunsire 's comment [in a MARC2RDA discussion](https://github.com/uwlib-cams/MARC2RDA/discussions/388#discussioncomment-5149681)...

> Seems that RDA Person is equivalent to Wikidata Natural Person; the "human" subclass has little semantic distinction.

...as an encouraging vote for using instances of non-RDA classes which can be shown to align with RDA Entities as needed to satisfy the domain/range requirements of RDA properties in well-formed RDA/RDF.

**Reply by GordonDunsire** [reply]
IRIs for instances of non-RDA classes may be used for "indirect conformance" with RDA, according to the guidance for [Conformance of metadata statements](https://access.rdatoolkit.org/en-US_topic_v3l_3wv_xmb/p_z2r_vyp_qlb)

The non-RDA class must be a sub-class of an RDA class:

IRI1 isA Z-work
Z-work subClassOf Work
=> IRI1 isA Work // well-formed RDA

Is schema:CreativeWork a sub-class of RDA Work?

The properties listed for [CreativeWork](https://schema.org/CreativeWork) indicate a conflation of RDA/LRM Work, Expression, and Manifestation. For example, schema:material and schema:provider are best mapped to RDA Manifestation properties. This implies that RDA Work, RDA Expression, and (some of) RDA Manifestation are sub-classes of schema:CreativeWork.

The only work-around that I know of is to map all of the schema properties associated with CreativeWork to RDA Work properties and to shortcuts involving RDA Expression and Manifestation properties. Also, some of the schema properties are accommodated by RDA representative expression properties.

For example, schema:material could be mapped to the shortcut chain:

Work has-manifestation-of-work Manifestation has-material ...

Confirming all of these mappings is a good mini-project***; I think this needs to be carried out before we can answer the question.

Is wd:Q17537576 "creative work" a sub-class of RDA Work?

The properties in WD indicate a typical (e.g. BIBFRAME) conflation of RDA/LRM Work and Expression. These properties can probably be readily mapped to RDA representative expression elements or shortcuts involving Work and Expression properties.

The answer is probably 'yes'.

Is wd:Q15621286 "intellectual work" a sub-class of RDA Work?

A general principle when mapping entities or classes is to map the broadest entity in the source scheme hierarchy as a sub-class of the the finest entity in the target scheme hierarchy; that is, to make the semantic gap as small as possible.

"intellectual work" is broader than (super-class of) "creative work", but the semantics of its properties seem to be fuzzier with respect to RDA/LRM. Many of the properties are shared, with a bias towards "creative work"; for example, wd:P170 "creator" is assigned to both classes, but is defined as "maker of this creative work or other object ...".

To play safe, my answer is 'no'.

Is wd:Q5 "human" a sub-class of RDA Person?

I would say 'yes'. The semantics are reasonably clear.

Applying the general principle, the broadest WD entity in the hierarchy that also fits is wd:Q154954 "natural person". So I would say that WD Natural Person is a sub-class of RDA Person, and that RDA Person is a sub-class of WD Natural Person.

The two statements indicate that the two classes are equivalent.

Note that the broader wd:Q3778211 "legal person" is too broad for RDA Person. Note also that WD says that "natural person" is "said to be the same as" "person", and maps "person" to external ontologies rather than "natural person". I'm not sure why ...

I think it is incumbent on the transformation project to provide such mappings if instance IRIs for non-RDA/LRM entities are used. It would be best to get the agreement of the other party, but life is too short.

In particular, I think it would be great if mappings for RDA entities were added to Wikidata; we can make this more formal via my membership of the RDA Technical Working Group by asking them to approve any proposed mappings.

*** Does anyone have access to a "study copy" of Ian Bigelow's "[Conducting the Opera: The Evolution of the RDA Work to the Share-VDE Opus and BIBFRAME Hub](https://link.springer.com/chapter/10.1007/978-3-031-16802-4_30)"? Or summarize it for me?

**Reply by briesenberg07** [reply]
Thank you @GordonDunsire --

I'm reflecting on [your response above](https://github.com/uwlib-cams/sinopia_maps/discussions/87#discussioncomment-5165481) and what seems an idea for a new/additional mapping project. 

@CECSpecialistI I'm wondering whether it would be appropriate to discuss this idea--that the use of resources typed with external classes in well-formed RDA/RDF should be supported by a published mapping of those classes to RDA/RDF--at MARC2RDA. I know that the group is limited in scope by necessity and most likely (99.9%) will *not* take on any additional mapping tasks, but this discussion would be regarding the necessity of such a mapping.

Or perhaps the use case for external classes other than those assigned to id.loc resources in RDA/RDF data is only a Sinopia resource templates issue, and such a meeting should be organized by me (I can't depend on MARC2RDA for *all* my answers!). In this case I would extend an invitation to Gordon and all MARC2RDA participants.

One final note about such a mapping is that the fact that Wikidata modeling (and possibly schema.org modeling as well? I'm not sure) is subject to change over time is somewhat scary to me.

\*\*\* Unfortunately Gordon I don't have access to [the article](https://link.springer.com/chapter/10.1007/978-3-031-16802-4_30) via UW Libraries! We don't seem to have the more recent years for those conference proceedings. Looks like $29.95 or bust for the time being.

**Reply by briesenberg07** [reply]
Thinking about how we could implement QA sources *now*, as the kind of mapping work described above has not been performed...

**Seems that there are some external classes which could be used relatively safely (we think) as values for RDA properties with non-WEMI RDA Entities as range.** 

**Using external classes as values for RDA properties with WEMI ranges seems not a best practice at this time.**

The upshot may be that creating W|E|M|I-to-W|E|M|I relationships in Sinopia will require describing a second (or third, etc.) resource.
